NCATS_Data_Dictionary.md

```diff
@@ -385,6 +385,8 @@ stakeholders.
     "TRIBAL", "TERRITORIAL", "PRIVATE")
 - `children` [list of strings]: Identifiers of organizations that are
   children of this organization
+- `enrolled` [ISO date]: Timestamp when this organization was enrolled
```

Comment:
I feel like it should be this but I see the document has similar field descriptions with and without the "of".

```suggestion
- `enrolled` [ISO date]: Timestamp of when this organization was enrolled
```

Reply:
I think either way is acceptable.  Just going to leave it as is, if that's alright with you.

Reply:
I think `Timestamp indicating when this organization was enrolled` is superior to the way it is now and the suggestion, but I don't care enough to cause a ruckus.

Reply:
Alright, you sold me on it.  Please see https://github.com/cisagov/ncats-data-dictionary/pull/44/commits/f0165745266aa2ffd125503fe2d8267323e78de8.

Reply:
~sold me~ strong-armed me :laughing:

Reply:
Yep. Sorry missed the update.